dada2_tutorial_16S.R

```diff
@@ -103,10 +103,10 @@ knitr::opts_chunk$set(eval = TRUE,
 #'
 #+ package installation, eval = FALSE, include=TRUE
 install.packages("BiocManager")
-BiocManager::install("dada2", version = "3.8")
+BiocManager::install("dada2", version = "3.14")
 
-source("https://bioconductor.org/biocLite.R")
-biocLite("ShortRead")
+# source("https://bioconductor.org/biocLite.R")
```

Comment:
Remove this source comment since we are no longer using biocLite?

Reply:
Agreed